inferno/lib/rule.py

```diff
@@ -201,6 +236,9 @@ def __init__(self,
         # input
         if isinstance(source_tags, basestring):
             source_tags = [source_tags]
+        if archive_lookback != 0:
+            source_tags = dateLookBack(source_tags, archive_lookback)
```

Comment:
The source tags part is not where this needs to go, checkout job.py and the archiver parts